mmr-testing/MMR-Test-InstanceOf-TimeRanges.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Test Instanceof timeranges</title>
+    <meta name="timeout" content="normal">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="mediasource-util.js"></script>
+</head>
+<body>
+    <div id="log"></div>
+    <script>
+        // Test Bug fix:
+        // Pull Request 7334313: Change to set TimeRangesProxy prototype to TimeRanges
+        function appendBuffer(test, sourceBuffer, data) {
+            test.expectEvent(sourceBuffer, "update");
+            test.expectEvent(sourceBuffer, "updateend");
+            sourceBuffer.appendBuffer(data);
+        }
+
+        var manifestFilename = "mp4/test-av-384k-44100Hz-1ch-320x240-30fps-10kfr-manifest.json";
+
+        mediasource_test(function (test, mediaElement, mediaSource) {
+            mediaElement.pause();
+           
+            mediaElement.addEventListener('error', test.unreached_func("Unexpected event 'error'"));
+
+            assert_true((mediaElement instanceof HTMLVideoElement), "mediaElement instanceof HTMLVideoElement");
+
+            //assert_true((mediaSource instanceof MediaSource), "mediaSource instanceof MediaSource");
```

Comment:
Does this also not work? (we can track with same bug as 40784788). Or should this not be commented out?

Reply:
No.. it was failing even before videElement is shimmed. so I will not enable it for now